test/controllers/herbaria_controller_test.rb

```diff
@@ -405,11 +407,9 @@ def test_create
                  herbarium.mailing_address)
     assert_equal(create_params[:description].strip, herbarium.description)
     assert_empty(herbarium.curators)
-    email = ActionMailer::Base.deliveries.last
-    assert_equal(katrina.email, email.header["reply_to"].to_s)
-    assert_match(/new herbarium/i, email.header["subject"].to_s)
-    assert_includes(email.body.to_s, "Burbank Herbarium")
-    assert_includes(email.body.to_s, herbarium.show_url)
+    assert_equal(count_before + 1, QueuedEmail.count)
+    assert_equal(katrina.id, QueuedEmail.last.user_id)
+    QueuedEmail.queue = false
```

Comment:
Hopefully this accomplishes what we want. The controller sends a webmaster email with `katrina` as the `user`.

Reply:
Whoops, just found the method `QueuedEmail#dump`. Will assert against that.

Reply:
Restored previous assertions using `QueuedEmail` methods.